### PR TITLE
updating reference to v2 and v3 of Extension bundles

### PR DESCRIPTION
**Pull request by v-smanchem**
Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

### PR information

- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 # Install extension bundles
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    EXTENSION_BUNDLE_VERSION_V2=2.13.0 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.14.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
     wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2/$EXTENSION_BUNDLE_FILENAME_V2 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2 && \
```
Comment (amamounelsayed) [inline host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile:24]: Is there a reason not to update v3 extension bundle?

File: host/4/bullseye/amd64/powershell/powershell72/powershell72-slim.Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 # Install extension bundles
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    EXTENSION_BUNDLE_VERSION_V2=2.13.0 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.14.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
     wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2/$EXTENSION_BUNDLE_FILENAME_V2 && \
```
Comment (amamounelsayed) [inline host/4/bullseye/amd64/powershell/powershell72/powershell72-slim.Dockerfile:23]: Is there a reason not to update v3 extension bundle?

File: host/4/bullseye/amd64/powershell/powershell70/powershell70.Dockerfile [hunk 3]
```diff
@@ -18,7 +18,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 # Install extension bundles
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    EXTENSION_BUNDLE_VERSION_V2=2.13.0 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.14.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
     wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2/$EXTENSION_BUNDLE_FILENAME_V2 && \
```
Comment (amamounelsayed) [inline host/4/bullseye/amd64/powershell/powershell70/powershell70.Dockerfile:23]: Is there a reason not to update v3 extension bundle?

File: host/4/bullseye/amd64/powershell/powershell70/powershell70-slim.Dockerfile [hunk 4]
```diff
@@ -18,7 +18,7 @@ RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
 # Install extension bundles
 RUN apt-get update && \
     apt-get install -y gnupg wget unzip && \
-    EXTENSION_BUNDLE_VERSION_V2=2.13.0 && \
+    EXTENSION_BUNDLE_VERSION_V2=2.14.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
```
Comment (amamounelsayed) [inline host/4/bullseye/amd64/powershell/powershell70/powershell70-slim.Dockerfile:22]: Is there a reason not to update v3 extension bundle?

File: host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile (hunk 1 above)
Comment (v-smanchem) [reply host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile:24]: There is no v3 extension bundle in the file you picked.

File: host/4/bullseye/amd64/powershell/powershell72/powershell72-slim.Dockerfile (hunk 2 above)
Comment (v-smanchem) [reply host/4/bullseye/amd64/powershell/powershell72/powershell72-slim.Dockerfile:23]: There is no v3 extension bundle in the file you picked.

File: host/4/bullseye/amd64/powershell/powershell70/powershell70.Dockerfile (hunk 3 above)
Comment (v-smanchem) [reply host/4/bullseye/amd64/powershell/powershell70/powershell70.Dockerfile:23]: There is no v3 extension bundle in the file you picked.

File: host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile (hunk 1 above)
Comment (v-smanchem) [reply host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile:24]: Working on it.

**Review by v-smanchem — COMMENTED** [review]
updated the files

File: host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile (hunk 1 above)
Comment (v-smanchem) [reply host/4/bullseye/amd64/powershell/powershell72/powershell72.Dockerfile:24]: Updated v3

**Review by vrdmr — APPROVED** [review]

**Pull request merged**